### PR TITLE
cloud: optimize 2 docs according to usability tests

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Fix 3 issues found in tests:

- User cannot connect to cluster if password is not set, but no such notice is given in the doc
- Sign in/out have unnecessary details.
- Role permissions might be better understood in a table.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### CODE REVIEW

File: tidb-cloud/manage-user-access.md [hunk 1]
```diff
@@ -145,21 +137,17 @@ If you are the owner of an organization, you can perform the following steps to
...
+|---------------------------------------------------------------------------------------|-------|--------|---------------|-------------|
+| Invite members to join the organization, and remove members from the organization | ✅     | ❌      | ❌             | ❌           |
+| Set roles for an organization member                                                  | ✅     | ❌      | ❌             | ❌           |
+| Create and rename projects                                                            | ✅     | ❌      | ❌             | ❌           |
+| Invite members to join a project, and remove members from a project                   | ✅     | ❌      | ❌             | ❌           |
+| Edit time zone                                                                        | ✅     | ❌      | ❌             | ❌           |
+| View bills and edit payment information                                               | ✅     | ❌      | ✅             | ❌           |
+| Join a project                                                                        | /     | ✅      | ✅             | ✅           |
```
Comment (ran-huang) [inline tidb-cloud/manage-user-access.md:148]:
```suggestion
| Join a project                                                                        | ✅    | ✅      | ✅             | ✅           |
```

File: tidb-cloud/manage-user-access.md [hunk 2]
```diff
@@ -145,21 +137,17 @@ If you are the owner of an organization, you can perform the following steps to
...
+| View and configure audit logging                                                      | ❌     | ❌      | ❌             | ✅           |
+| Obtain project instance management rights                                             | /     | ✅      | ✅             | ✅           |
```
Comment (ran-huang) [inline tidb-cloud/manage-user-access.md:150]:
```suggestion
| Obtain project instance management rights                                             | ✅     | ✅      | ✅             | ✅           |
```

File: tidb-cloud/manage-user-access.md (hunk 1 above)
Comment (ran-huang) [reply tidb-cloud/manage-user-access.md:148]: "Join a project" and "Obtain project instance management rights" are not stated for the **owner** role. I'm not sure if they apply to owner.

File: tidb-cloud/manage-user-access.md [hunk 3]
```diff
@@ -11,19 +11,11 @@ This document describes how to manage user access of the [TiDB Cloud console](ht
 
 1. Navigate to the TiDB Cloud login page: <https://tidbcloud.com>.
```
Comment (qiancai) [inline tidb-cloud/manage-user-access.md:12]:
```suggestion
1. Go to the TiDB Cloud login page: <https://tidbcloud.com>.
```

File: tidb-cloud/manage-user-access.md [hunk 4]
```diff
@@ -11,19 +11,11 @@ This document describes how to manage user access of the [TiDB Cloud console](ht
...
+2. On the TiDB Cloud login page, sign in with Google, GitHub, or email.
 
 ## Sign out
 
-After you have signed into TiDB Cloud, if you need to sign out, perform the following steps:
-
-1. Click the account name on the upper right of the window.
-
-2. Click **Logout**.
+To sign out from TiDB Cloud, click your account name and select Logout.
```
Comment (qiancai) [inline tidb-cloud/manage-user-access.md:18]:
```suggestion
In the upper-right corner of the TiDB Cloud console, click your account name and select **Logout**.
```

File: tidb-cloud/manage-user-access.md (hunk 1 above)
Comment (qiancai) [reply tidb-cloud/manage-user-access.md:148]: I think it is both yes. Owners can join a project owned by others and obtain project instance management rights as well. It seems that we also need a technical review of this table.

File: tidb-cloud/manage-user-access.md [hunk 5]
```diff
@@ -145,21 +137,17 @@ If you are the owner of an organization, you can perform the following steps to
 
 There are four roles in an organization. The permissions of each role are as follows:
```
Comment (ran-huang) [inline tidb-cloud/manage-user-access.md:139]: @langyuezhang PTAL

File: tidb-cloud/manage-user-access.md [hunk 6]
```diff
@@ -145,21 +137,17 @@ If you are the owner of an organization, you can perform the following steps to
...
+| Set roles for an organization member                                                  | ✅     | ❌      | ❌             | ❌           |
+| Create and rename projects                                                            | ✅     | ❌      | ❌             | ❌           |
+| Invite members to join a project, and remove members from a project                   | ✅     | ❌      | ❌             | ❌           |
+| Edit time zone                                                                        | ✅     | ❌      | ❌             | ❌           |
+| View bills and edit payment information                                               | ✅     | ❌      | ✅             | ❌           |
+| Join a project                                                                        | ✅    | ✅      | ✅             | ✅           |
+| View and configure audit logging                                                      | ❌     | ❌      | ❌             | ✅           |
+| Obtain project instance management rights                                             | ✅     | ✅      | ✅             | ✅           |
```
Comment (ran-huang) [inline tidb-cloud/manage-user-access.md:150]: @langyuezhang PTAL

File: tidb-cloud/manage-user-access.md [hunk 7]
```diff
@@ -145,21 +137,17 @@ If you are the owner of an organization, you can perform the following steps to
...
+|---------------------------------------------------------------------------------------|-------|--------|---------------|-------------|
+| Invite members to join the organization, and remove members from the organization | ✅     | ❌      | ❌             | ❌           |
+| Set roles for an organization member                                                  | ✅     | ❌      | ❌             | ❌           |
+| Create and rename projects                                                            | ✅     | ❌      | ❌             | ❌           |
+| Invite members to join a project, and remove members from a project                   | ✅     | ❌      | ❌             | ❌           |
+| Edit time zone                                                                        | ✅     | ❌      | ❌             | ❌           |
+| View bills and edit payment information                                               | ✅     | ❌      | ✅             | ❌           |
+| Join a project                                                                        | ✅    | ✅      | ✅             | ✅           |
```
Comment (ran-huang) [inline tidb-cloud/manage-user-access.md:148]:
```suggestion
```

File: tidb-cloud/manage-user-access.md [hunk 8]
```diff
@@ -168,21 +160,16 @@ If you are the owner of an organization, you can perform the following steps to
...
-    - View bills and edit payment information
-- Member:
-    - Can be invited to join a project and obtain project instance management rights
-- Billing Admin:
-    - View bills and edit payment information
-    - Can be invited to join a project and obtain project instance management rights
-- Audit Admin:
-    - View and configure audit logging
-    - Can be invited to join a project and obtain project instance management rights
+|                                                                                       | Owner | Member | Billing Admin | Audit Admin |
```
Comment (ran-huang) [inline tidb-cloud/manage-user-access.md:163]:
```suggestion
|  Permission                                                                        | Owner | Member | Billing Admin | Audit Admin |
```

**Review by qiancai — APPROVED** [review]
LGTM

File: tidb-cloud/manage-user-access.md [hunk 9]
```diff
@@ -168,21 +160,16 @@ If you are the owner of an organization, you can perform the following steps to
...
+| Invite members to join the organization, and remove members from the organization | ✅     | ❌      | ❌             | ❌           |
+| Set roles for an organization member                                                  | ✅     | ❌      | ❌             | ❌           |
+| Create and rename projects                                                            | ✅     | ❌      | ❌             | ❌           |
+| Invite members to join a project, and remove members from a project                   | ✅     | ❌      | ❌             | ❌           |
+| Edit time zone                                                                        | ✅     | ❌      | ❌             | ❌           |
+| View bills and edit payment information                                               | ✅     | ❌      | ✅             | ❌           |
+| View and configure audit logging                                                      | ❌     | ❌      | ❌             | ✅           |
+| Obtain project instance management rights                                             | ✅     | ✅      | ✅             | ✅           |
```
Comment (ran-huang) [inline tidb-cloud/manage-user-access.md:172]:
```suggestion
| Obtain project instance management rights                                             | ✅     | ✅      | ✅             | ✅           |
| Create API key                                                                                              | ✅     | ❌      | ❌             | ❌            |
```

File: tidb-cloud/manage-user-access.md [hunk 10]
```diff
@@ -168,21 +160,17 @@ If you are the owner of an organization, you can perform the following steps to
...
+| Create an API key                                                                                              | ✅     | ❌      | ❌             | ❌            |
```
Comment (ran-huang) [inline tidb-cloud/manage-user-access.md:173]:
```suggestion
| Manage an API key                                                                                              | ✅     | ❌      | ❌             | ❌            |
```

**Pull request merged**